torch/utils/cpp_extension.py

```diff
@@ -284,25 +284,26 @@ def _join_sycl_home(*paths) -> str:
 
 _COMMON_SYCL_FLAGS = [
     '-fsycl',
-    '-fsycl-targets=spir64_gen,spir64',
 ]
 
 def _get_sycl_arch_list():
-    if 'TORCH_XPU_ARCH_LIST' in os.environ:
```

Comment:
@dvrogozh Do you have any comments if we remove this fast path?

Reply:
It will take the aot targets which the pytorch binary was compiled against. Removing this line just disables the possibility of customizing AOT targets when building an extension. If you think we need the AOT target customization capability, I can add the line back.

Reply:
> Removing this line just disables the possibility of customizing AOT targets when building an extension.

That's why this line was added! yes, I do believe that users must have a way to customize arch list for which extension is built. Take care to review the whole cpp_extension.py file - for CUDA there is the similar code. **Please, do get this variable back.**